tests/helpers.py

```diff
@@ -380,21 +388,34 @@ def delete_cephblockpool():
         kind=constants.CEPHBLOCKPOOL,
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
-    pool_list = get_cephblockpool_name()
-    for item in pool_list:
-        logger.info(f"Deleting CephBlockPool with name {item}")
-        assert POOL.delete(resource_name=item)
+    logger.info(f"Deleting CephBlockPool with name {cbp_name}")
+    assert POOL.delete(resource_name=cbp_name)
     return True
 
 
-def create_cephfilesystem():
+def create_cephfilesystem(override_fs=True):
     """
     Function for deploying CephFileSystem (MDS)
 
+    Args:
+        override_fs (bool): If true it will delete old CFS and recreate new one, If False will skip creation of CFS
+
     Returns:
         bool: True if CephFileSystem creates successful
     """
     fs_data = copy.deepcopy(defaults.CEPHFILESYSTEM_DICT)
+    if override_fs:
+        logger.info("Deleting CephFileSystem if exists")
+        delete_all_cephfilesystem()
```

Comment:
Should `delete_all_cephfilesystem` get the same treatment as other helpers and change from removing "all" of a resource to one specific resource? I think consistency would be better here.

Reply:
Done made a function that gets the cephfs name and passes it to delete_cephfilesystem() renamed delete_all_cephfilesystem to delete_cephfilesystem